Makefile

```diff
@@ -69,10 +76,26 @@ fix: ## Fix linting issues in the code
 	${DOCKER_COMPOSE} run --rm app npm run lint:fix
 
 test: export TARGET = dev
-test: ## Run the tests
-	${DOCKER_COMPOSE} run --rm app npm run test
+test: ## Run all the tests
+	$(MAKE) initdb
+	${DOCKER_COMPOSE} run --rm app npm run test; ${DOCKER_COMPOSE} down
+
+unit-test: export TARGET = dev
+unit-test: ## Run the unit tests
+	${DOCKER_COMPOSE} run --rm app npm run test:unit
+
+integration-test: export TARGET = dev
+integration-test: ## Run the integration tests
+	$(MAKE) initdb
+	${DOCKER_COMPOSE} run --rm app npm run test:integration; ${DOCKER_COMPOSE} down
```

Comment:
Doesn't this make the exit code the one from `docker-compose down`?

Reply:
Nope.

Reply:
This test run should fail but without the exit code capture it just exits with a `0`

Reply:
Huh, locally I get a `2`.

Reply:
Oh no, I don't for that one... Tried something similar earlier and it worked? 🤔